### PR TITLE
add option to set cacheDir

### PR DESCRIPTION
**Pull request by joeybaker**
I'd like to store the cache outside of `node_modules`. his provides an easy way to configure the directory the cache is saved to.

### CODE REVIEW

File: bin/cmd.js [hunk 1]
```diff
@@ -11,7 +11,8 @@ function run() {
   var _argv = process.argv.slice( 2 );
   var persistifyArgs = subarg( _argv, {
     alias: {
-      'n': 'never-cache'
+      'n': 'never-cache',
+      'd': 'cache-directory'
```
Comment (royriojas) [inline bin/cmd.js]: shouldn't this be `cache-dir` ?

File: bin/cmd.js [hunk 2]
```diff
@@ -19,9 +20,11 @@ function run() {
   var recreate = persistifyArgs.recreate;
   var neverCache = persistifyArgs[ 'never-cache' ];
   var cacheId = persistifyArgs[ 'cache-id' ];
+  var cacheDir = persistifyArgs[ 'cache-directory' ];
```
Comment (royriojas) [inline bin/cmd.js]: I guess this should be `cache-dir`, to match the info in the readme

File: bin/cmd.js (hunk 1 above)
Comment (joeybaker) [inline bin/cmd.js]: Whoops, you're right. Which do you prefer?

Comment (royriojas) [inline bin/cmd.js]: `cache-dir`. it is shorter.

**Comment by joeybaker** [comment]
@royriojas sorry for my stupid. Updated now!

**Comment by royriojas** [comment]
published `v0.3.3`

**Comment by joeybaker** [comment]
Thanks @royriojas!

**Pull request merged**